lib/include/openamp/virtqueue.h

```diff
@@ -69,6 +78,7 @@ struct virtqueue {
 	uint16_t vq_free_cnt;
 	uint16_t vq_queued_cnt;
 	void *shm_io; /* opaque pointer to data needed to allow v2p & p2v */
+	struct metal_list bounce_buf_list;
```

Comment:
Fails if HVL_VIRTIO is not defined, please put under #if defined(HVL_VIRTIO)

Reply:
Will move to new PR for hypervisorless virtio